src/khoj/search_type/text_search.py

```diff
@@ -208,7 +208,7 @@ def setup(
     text_to_entries: Type[TextToEntries],
     files: dict[str, str],
     regenerate: bool,
-    user: KhojUser = None,
+    user: KhojUser,
```

Comment:
Did you want to move the `user` parameter to the top, similar to the other functions?

Reply:
Yes

Reply:
I'll let this be as is for now, don't want to shuffle too much code around as part of this change